bootloader/src/pyi_utils.c

```diff
@@ -246,10 +271,32 @@ pyi_get_temp_path(char *buffer)
         if (_wmkdir(wchar_ret) == 0) {
             pyi_win32_utils_to_utf8(buffer, wchar_ret, PATH_MAX);
             free(wchar_ret);
+            if (NULL != runtime_tmpdir) {
+              /*
+               * Restore TMP to what it was
+               */
+              if (NULL != original_tmpdir) {
+                _wputenv_s(L"TMP", original_tmpdir);
+                free(original_tmpdir);
+              } else {
+                _wputenv_s(L"TMP", L"");
```

Comment:
Please use `pyi_unsetenv`, since if the variable was not set earlier, it should not be set now.